ios/CollectorManager.swift

```diff
@@ -15,6 +15,30 @@ class CollectorManager: NSObject {
         
     CollectorManager.map[name] = VGSCollect(id: vaultId, environment: environment);
  }
+  
+    @objc(pinConfirm:path:method:headers:resolver:rejecter:)
+    func pinConfirm(_ name: String, path: String, method: String, headers: Dictionary<String, String>, resolver resolve: @escaping RCTPromiseResolveBlock, rejecter reject: @escaping RCTPromiseRejectBlock) -> Bool {
+      let collector = CollectorManager.map[name]!
+        
+      if (collector == nil) {
+          reject("error", "invalid name", NSError(domain: "MissingCollectorByNameError", code: 401))
+          return;
+      }
+      collector.customHeaders = headers;
+
+      let pin = collector.getTextField(fieldName: "data.attributes.pin")
```

Comment:
I would say that field names should be dynamic and not a fixed value

Reply:
@JorgeDelgadillo Just take into account this comment [here](https://github.com/Instawork/mobile/pull/4865#discussion_r1106123340)